iOSNativeLoginTemplate/iOSNativeLoginTemplate/SceneDelegate.swift

```diff
@@ -129,3 +163,28 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
        postResetBlock()
    }
 }
+
+@MainActor class ReCaptchaClientObservable: ObservableObject {
+    
+    @Published var reCaptchaClient: RecaptchaClient? = nil
+    
+    init(reCaptchaSiteKey: String) {
+        Task(priority: .medium) {
+            await initializeReCaptchaClient(reCaptchaSiteKey: reCaptchaSiteKey)
+        }
+    }
+    
+    final func initializeReCaptchaClient(reCaptchaSiteKey: String) async {
+        do {
+
+            reCaptchaClient = try await Recaptcha.getClient(withSiteKey: reCaptchaSiteKey)
+        } catch let error {
+            print("Cannot get reCAPTCHA client due to an error with description '\(error.localizedDescription).'.")
+        }
+    }
+}
+
+class NavigationPathObservable: ObservableObject {
+    
+    @Published var navigationPath: [NavigationDestination] = []
```

Comment:
In the context of a full app, I'd love to see how this turned into popping in and back out of deep links.